spec.bs

```diff
@@ -267,7 +260,7 @@ This document introduces a new storage API that is intentionally not partitioned
 
 The {{SharedStorageWorklet}} Interface {#worklet}
 =================================================
-The {{SharedStorageWorklet}} object allows developers to supply [=module scripts=] to process [=Shared Storage=] data and then output the result through one or more of the output gates. Currently there are two output gates, the [=Private Aggregation=] output gate and the {{SharedStorageWorklet/selectURL()|URL-selection}} output gate.
+The {{SharedStorageWorklet}} object allows developers to supply [=module scripts=] to process [=Shared Storage=] data and then output the result through one or more of the output gates. Currently there are two output gates, the <code>{{privateAggregation}}.{{PrivateAggregation/contributeToHistogram()}}</code> output gate and the {{SharedStorageWorklet/selectURL()|URL-selection}} output gate.
```

Comment:
I understand the desire to link to a definition, but we're introducing contributeToHistogramOnEvent(), so I think it might be better to just link to the spec overall. We could just do that with an <a> tag, though.

Reply:
Works for me :)

Reply:
I went with the normative reference syntax (`[[!FOO]]`).

Reply:
Looks like this was undone in the latest force-push

Reply:
Good catch! Fixed.